lib/stm32/common/exti_common_all.c

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library.  If not, see <http://www.gnu.org/licenses/>.
  *
- * This provides the code for the "next gen" EXTI block provided in F2/F4/L1
+ * This provides the code for the "next gen" EXTI block provided in F2/F4/F7/L1
```

Comment:
If you want to touch this line, you'd be safer saying, "non-f1" rather than leaving it still incomplete. (l0,l4, f3, f0...)

though, in _reality_ this file is actually included for f1 too, so this was just badly copied from somewhere else in the past.